fiftyone/core/clips.py

```diff
@@ -964,8 +968,9 @@ def _write_manual_clips(dataset, src_collection, clips, other_fields=None):
             ]
         )
     finally:
-        cleanup_op = {"$unset": {_tmp_field: ""}}
-        src_dataset._sample_collection.update_many({}, cleanup_op)
+        src_dataset._sample_collection.update_many(
+            {}, {"$unset": {_tmp_field: ""}}
```

Comment:
Same here

Reply:
`$set` and `$unset` are okay in update operations, they just cannot be used in aggregation pipelines in DocumentDB/older Mongo versions

Reply:
Ok I did not notice it was an update op